### PR TITLE
REX-Ray Docker Plug-in for DigitalOcean Block Storage

### LINKED ISSUE #816: Create plugin for Digital Ocean
# Summary
Create a Docker managed volume plugin for Digital Ocean from existing `dobs` storage driver.

### PR DESCRIPTION
**Pull request by cduchesne**
This patch provides a REX-Ray Docker plugin for DigitalOcean Block Storage

Fixes #816

### CODE REVIEW

File: .docs/user-guide/docker-plugins.md [hunk 1]
```diff
@@ -243,6 +243,44 @@ Environment Variable | Description | Default | Required
...
+
+Type | Value
+-----|------
+network | `host`
+mount | `/dev`
+allow-all-devices | `true`
+capabilities | `CAP_SYS_ADMIN`
+
+#### Configuration
+The following environment variables can be used to configure the GCEPD
```
Comment (codenrhoden) [inline .docs/user-guide/docker-plugins.md]: s/GCEPD/DOBS

**Comment by codenrhoden** [comment]
Ah, one more thing...  does `REXRAY_BUILD_TYPE= DRIVERS=dobs` need to be added to `.travis.yml` still? I think it does.

**Comment by akutz** [comment]
It does, and @cduchesne noted that in Slack in the last few hours.

**Comment by codenrhoden** [comment]
So the change to the Travis file should be a part of this PR, right?

**Pull request merged**